### ISSUE TITLE
ACL support

### ISSUE THREAD

**Issue by MrSandmanRUS** [ytsaurus/ytsaurus-spyt#14]
At the moment, when launching spyt-job, it is impossible to specify which other users can work with the Job in order to view logs, be able to kill the Job, and so on.

It will be great to be able to specify ACL settings when spark-submit.

**Comment by MrSandmanRUS** [comment]
I am trying spark-submit --master ytsaurus://$YT_PROXY ...

Now I have not found an opportunity to transfer the ACL.

**Comment by Alexvsalexvsalex** [comment]
It exists. It's available in the last release 2.1.0.
Please, look here: https://ytsaurus.tech/docs/en/user-guide/data-processing/spyt/thesaurus/gpu-usage#configuring-the-gpu-for-direct-execution. There is example of redefining `pool` using `spark.ytsaurus.executor.operation.parameters`. You can write kind of:
```
spark-submit ... --conf spark.ytsaurus.driver.operation.parameters="{acl=[...]}" --conf spark.ytsaurus.executor.operation.parameters="{acl=[...]}"
```

**Comment by MrSandmanRUS** [comment]
This is great news, apparently I missed it, thanks for the link. We will try to update our version.
